### DISCUSSION TITLE
Installation guidance for an existing Dudende installation

### DISCUSSION THREAD

**Discussion by jcnewman83** [skoruba/Duende.IdentityServer.Admin#48] [Q&A]
Hi,

I have an existing Duende installation that I want to be able to manage using Skoruba ID Admin, however, I am looking for a little guidence.

I want to have the admin installation in a new web project away from my duende install so, I followed the steps to start a new project, made changes to the DB connection strings in the appconfig and started up the project, however, the project failed to start because it failed to apply the DB migrations which makes sense because having reviewed the migrations it looks like they include schema that ships with the Duende install.

So onto my question :-)
What do you recommend are the best steps for me to get Skoruba up and running given that I have a pre-existing installation of Duende?

**Comment by skoruba** [comment]
Hi @jcnewman83 - thank you for your question - this is in my todo list, but I am busy now, I will prepare some guide how to install it ASAP. 😊

**Reply by jcnewman83** [reply]
Thank you @skoruba I really appreciate your help.

**Reply by altso** [reply]
@skoruba did you finish the guide? Having exactly the same question as @jcnewman83

**Reply by DustinReagan** [reply]
I have the same situation, is this guide available?  Thanks!

**Reply by keithswright** [reply]
Similar issue here. 

After hours of reading through old bugs and pull requests about splitting the UI our from the core code, I believe it was possible to just add the UI to a .net core web application with identity server 4.  https://github.com/skoruba/IdentityServer4.Admin/pull/779
So i believe it should be possible with duende too, but I cant find any documentation on how to do it. So I tried adding the all the nuget packages manually and doing this: 

        builder.Services.AddIdentityServerAdminUI<ApplicationDbContext, IdentityServerConfigurationDbContext, IdentityServerPersistedGrantDbContext, AdminLogDbContext, AdminAuditLogDbContext, AuditLog, IdentityServerDataProtectionDbContext, ApplicationUser>(options =>
        {
                // Applies configuration from appsettings.
                options.BindConfiguration(builder.Configuration);
                if (builder.Environment.IsDevelopment())
                {
                    options.Security.UseDeveloperExceptionPage = true;
                }
                else
                {
                    options.Security.UseHsts = true;
                }

                // Set migration assembly for application of db migrations
                options.DatabaseMigrations.SetMigrationsAssemblies(migrationsAssembly);

                // Use production DbContexts and auth services.
                options.Testing.IsStaging = false;
        });

But I'm still getting runtime errors at startup. So, if there was a guide coming, it would be much appreciated.

**Reply by alex-pollan** [reply]
Any update on this one guys? I have a similar requirement. Thanks!

**Reply by skoruba** [reply]
Sorry for delay, I am working on it. :)

**Comment by tkardaridis** [comment]
Any update on this? 
I have the same requirement. I have a custom Identity Server implementation with Angular App for the login and logout and I need the AdminUI to manage the server.  @skoruba did you finish the quide? 

Regards

Thomas

**Reply by skoruba** [reply]
Hi @tkardaridis - still working on it. I will send an update soon! Thanks!

**Comment by clint2627** [comment]
Is a guide available? I am trying to accomplish the same thing and I'm worried about what the database migrations will do. I would assume on an existing database we don't need to execute the migrations?
